### PR TITLE
UCR: Only show pagination warning on map reports

### PR DESCRIPTION
**Pull request by NoahCarnahan**
@kaapstorm 
The pagination warning shows on other types of reports, which I don't think was intentional.

### CODE REVIEW

**Comment by kaapstorm** [comment]
:+1: Thanks, Noah.

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 1]
```diff
@@ -59,11 +59,17 @@
         };
 
         var paginationNotice = function (data) {
-            if (data.aaData.length < data.iTotalRecords) {
-                $('#info-message').html("{% trans 'Showing the current page of data. Switch pages to see more data.' %}");
-                $('#report-info').removeClass('hide');
-            } else {
-                $('#report-info').addClass('hide');
+            if (mapSpec) {  // Only show warning for map reports
+                if (data.aaData !== undefined && data.iTotalRecords !== undefined) {
```
Comment (millerdev) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: nit: could combine these into a single `if` block using  using `&&`

File: corehq/apps/reports_core/templates/reports_core/base_template_new.html [hunk 2]
```diff
@@ -59,11 +59,17 @@
...
+                    if (data.aaData.length < data.iTotalRecords) {
```
Comment (millerdev) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: Looks like this is controlling a visibility toggle of `#report-info`. Should `#report-info` be hidden if `!mapSpec || data.aaData === undefined || data.iTotalRecords === undefined`, or is it intended to not modify `#report-info` in any way if that's true?

Comment (NoahCarnahan) [inline corehq/apps/reports_core/templates/reports_core/base_template_new.html]: Correct, it's not intended to not modify `#report-info` if that condition is not true.

**Pull request merged**